osu.Game.Rulesets.Taiko/Mods/TaikoModConstantSpeed.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Taiko.Objects;
+using osu.Game.Rulesets.Taiko.UI;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.UI;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModConstantSpeed : Mod, IApplicableToDrawableRuleset<TaikoHitObject>
+    {
+        public override string Name => "Constant Speed";
+        public override string Acronym => "CS";
+        public override double ScoreMultiplier => 0.8;
```

Comment:
Any reason for this being 0.8x while it's 0.9x in osu!mania?

Reply:
i've asked around a bunch of osu!taiko mappers and players, and suggestions varied from 0.7 to 0.9, so i've settled with the middle.

I don't mind changing to 0.9 for consistency tho.

Reply:
I'd change it yes